content/02.intro.md

```diff
@@ -1,27 +1,27 @@
 ## Introduction {#intro}
 
-Machine learning is a branch of computational science organized around the learning by computers via algorithms of relationships between dataset variables. Traditionally this has referred to statistical methods such as linear and logistic regression, but as the scale and complexity of data have increased, many new ML methods have been developed to model and learn increasingly complex relationships.
-One powerful approach for learning complex (that is, both linear and non-linear) relationships between input and output variables is called an artificial neural network.
-These networks are modeled after the human brain, and they comprise artificial neurons arranged into layers.
-Each layer receive input from previous layers (the first of which represents the input data), and then "fires off" its own weighted output that then serves as input into subsequent layers in the network. 
-Thus, the process of "training" a neural network is in fact the tuning of these output weights such that a metric called a cost function is optimized.
-Deep learning utilizes artificial neural networks with many layers (hence the term "deep"), and given the computational advances of recent decades, is now capable of being applied to massive data sets and in enumerable contexts.
-In many circumstances, deep learning has the capacity to learn more complex relationships and make more accurate predictions than other methods.
-Therefore, deep learning has become its own subfield of machine learning, and in the context of biological research, has been increasingly used to derive novel insights from high-dimensional biological data [@doi:10.1098/rsif.2017.0387].
+Machine learning is a modern approach to problem-solving and task automation. In particular, machine learning is concerned with the development and applications of algorithms that learn how to recognize patterns in data and utilize these for predictive modeling, as opposed to having domain experts developing rules for prediction tasks manually.
+Artificial neural networks are a particular class of machine learning algorithms and models that evolved into what we now describe as "deep learning" -- that is, neural networks with many layers (and algorithms that make them perform well).
+These neural networks comprise artificial neurons arranged into layers and are modeled after the human brain, even though the building blocks and learning algorithms may differ [@doi:10.1038/s41583-020-0277-3].
+Each layer receives input from previous layers (the first of which represents the input data) and then transmits a weighted version of its input to the subsequent layer. 
+Thus, the fitting or process of "training" a neural network is the tuning of the layers' weights such that minimizes a so-called cost or loss function that serves as a differentiable surrogate of the prediction error.
```

Comment:
Suggested edit-

Thus, the process of "training" a neural network is the tuning of the layers' weights to minimize a cost or loss function that serves as a differentiable surrogate of the prediction error.

Reply:
I like it.